### ISSUE TITLE
shareImage and thumbnail seems not working when sharing on Twitter

### ISSUE THREAD

**Issue by rverchere** [chipzoller/hugo-clarity#332]
**I have checked all the prerequisites below and I'm yet experiencing a problem**

- [X] Read the README.md
- [X] Have the extended version of Hugo installed
- [X] Used the exampleSite's config.toml as a reference
- [X] I have included my repository link. Instead of pasting long snippets here.

**Describe the bug**

Hi, when sharing an article on Twitter, the image (shareImage or thumbnail) is not embedded in the link.
There is the same issue in your official blog neonmirrors.net.

**Desktop (please complete the following information):**
 - OS: GNU/Linux Mint
 - Browser Brave Version 1.39.122 Chromium: 102.0.5005.115
 -

**Comment by chipzoller** [comment]
This was very recently fixed if you can pull and update from Master.

**Comment by rootwork** [comment]
Yeah it looks like there is something wrong going on here. The OpenGraph image URL being served on that blog post is `https://neonmirrors.net/post/2022-06/examining-pod-security/images/2022-06/examining-pod-security/thumbnail.png`.

My guess is LinkedIn just does a better job falling-back if the OG image returns a 404, and goes and searches the page for an image to use.

**Comment by chipzoller** [comment]
like I said, it should be fixed in master right now. I haven't updated Clarity to include that code which is why the bug is still present.

**Comment by rverchere** [comment]
> like I said, it should be fixed in master right now. I haven't updated Clarity to include that code which is why the bug is still present.

Before answering I've updated the master branch on my personal blog ( hugo + submodule theme @ [6347e453](https://github.com/chipzoller/hugo-clarity/tree/6347e453406917e5c9c99b3af2a2eacb4a407b8b) ) and I still have the issue.

**Comment by rootwork** [comment]
Hmm. That is strange. Running it through the [Twitter Card Validator](https://cards-dev.twitter.com/validator) confirms this.

The only possible issue I can see is the lack of a space before the closing `/>` on [that line in the partial](https://github.com/chipzoller/hugo-clarity/blob/master/layouts/partials/opengraph.html#L57). It's hard for me to believe Twitter would choke on something like that, but it'd be easy enough to fix, so let me try it.

**Comment by rverchere** [comment]
I'm trying to find differences from other working sites.
Here there is no `og:image:width`, `og:image:height` and `og:image:type` meta properties.

**Comment by rverchere** [comment]
fyi, I've set the `largeTwitterCard` to true, and now the image is available in tweet cards.

**Comment by rverchere** [comment]
Deleted `largeTwitterCard`, still working. I guess there were some cache when I upgraded to the master branch.

You can close the issue.
